charts/importer/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: importer
-version: 1.1.3
```

Comment:
Please bump this to 1.2.0 as it adds a new feature :)

Reply:
I was hesitant on that one, changes done :)